src/main/java/uk/nhs/digital/nhsconnect/nhais/service/edifact_to_fhir/GenderPatchTransactionMapper.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.digital.nhsconnect.nhais.service.edifact_to_fhir;
+
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Transaction;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.AmendmentPatch;
+import uk.nhs.digital.nhsconnect.nhais.model.jsonpatch.JsonPatches;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class GenderPatchTransactionMapper implements PatchTransactionMapper{
+
+    @Override
+    public AmendmentPatch map(Transaction transaction) {
+        var gender = transaction.getGender();
+        if (gender.isPresent()) {
+            var genderName = gender.get().getGender().getName().toLowerCase();
```

Comment:
Need to check that `gender.get().getGender().getName()` are actually the gender enumerated values supported by FHIR

Reply:
PersonSex class has a FROM_FHIR_MAP. I think we should do the inverse (TO_FHIR_MAP) and then use Enumerations.AdministrativeGender.toCode() to ensure that we have the correct FHIR value here.

Reply:
done